hr_holidays_type_note/views/hr_leave.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_leave_form_note" model="ir.ui.view">
+        <field name="name">hr.leave.form.note</field>
```

Comment:
Please use the schema `modulname.id_of_inherited_view`. In this case it is `hr_holidays_type_note.hr_leave_view_form`. Got it?

Reply:
I will make sure to use this naming schema for all future modules